example/java/build_library.sh

```diff
@@ -14,6 +32,13 @@ elif [[ "$unamestr" == 'Darwin' ]]; then
     cp build/shared_model/bindings/libirohajava.jnilib dist/libirohajava.jnilib
 fi
 
+if [[ $# == 1 ]]; then
+    #make java package folder path(org.iroha to org/iroha)
```

Comment:
if the package contains more than 2 elems (e.g: _org.iroha.my.proj_)? use sed instead